ext/reflection/php_reflection.stub.php

```diff
@@ -638,8 +638,7 @@ class ReflectionParameter implements Reflector
     /** @implementation-alias ReflectionClass::__clone */
     private function __clone(): void {}
 
-    /** @param string|array|object $function */
-    public function __construct($function, int|string $param) {}
+    public function __construct(string|array|object $function, int|string $param) {}
```

Comment:
Is https://github.com/php/php-src/blob/2501cad25a1818fa35830982371ae88b0adb5d57/ext/reflection/php_reflection.c#L2537 dead code then?

Reply:
I don't think so, since it is still reached in https://github.com/php/php-src/blob/2501cad25a1818fa35830982371ae88b0adb5d57/ext/reflection/tests/ReflectionParameter_invalidMethodInConstructor.phpt#L32-L47

The parameter is still parsed via `Z_PARAM_ZVAL`, rather than enforcing the types as part of the parsing